ConnectorDynamo/ConnectorDynamoFunctions/BatchConverter.cs

```diff
@@ -226,7 +226,7 @@ public object ConvertDataTreeToNative(Base @base)
         .Split(';') // Split by ;
         .Select(text =>
         {
-          int.TryParse(text, out var num);
+          _ = int.TryParse(text, out var num); // At this point, we expect split to yield all integers based on the dataTreePathRegex check above.
```

Comment:
I think unless the comment is short, be a bit more readable before the method

Reply:
Modified in latest commit